harmony_netcdf_to_zarr/download_utilities.py

```diff
@@ -55,14 +55,21 @@ def download_granules(netcdf_urls: List[str], destination_directory: str,
             download_process.start()
 
         # Ensure worker processes exit successfully
+        exit_codes = []
         for download_process in processes:
             download_process.join()
+            exit_codes.append(download_process.exitcode)
             download_process.close()
 
+        logger.info(f'process exitcodes: {exit_codes}')
```

Comment:
Is this debug logging from development?

Reply:
It wasn't, but I can see it's not needed since they will be zeros or an exception will be raised